ad/movement/trusts.md

```diff
@@ -477,6 +544,11 @@ When an ADCS is installed and configured in an Active Directory environment, a C
 
 // group scoping, [https://posts.specterops.io/a-pentesters-guide-to-group-scoping-c7bbbd9c7560](https://posts.specterops.io/a-pentesters-guide-to-group-scoping-c7bbbd9c7560)
 
+#### Abusing Shadow Principals
+1. If one compromises an account in the Shadow Principals, they can use the privileges to access the user forest using RDP (explicit credentials of the bastion user required), WMI, PowerShell Remoting etc. It is important to note that if Kerberos AES Encryption is not enabled for the PAM trust, one needs to add the machines of the existing forest in `WSMan TrustedHosts` and use the `-Authentication Negotiate` option with PowerShell remoting cmdlets.
+2. It is also possible to use the `SIDHistory` injection attack using `Mimikatz` to abuse the PAM trust.
+3. Overall, auditors should look for a way to compromise the Bastion (Red) forest to easily gain access to all the managed Production Forests.
+
```

Comment:
Removed this as it was pretty much redundant. In the end, Bastion Forests are one-way trusted forests, no special attacks for them 🤷

Reply:
I can't argue with that, the enumeration part is what differentiates those attacks.